dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/GenericFilter.java

```diff
@@ -219,6 +228,54 @@ private String getGenericValueFromRpcContext() {
         return generic;
     }
 
+    public Method findMethodByMethodSignature(Class<?> clazz, String methodName, String[] parameterTypes, ServiceModel serviceModel)
```

Comment:
Is this just a copy of findMethodByMethodSignature with only different parameters?

Reply:
Yes